src/app/views/App.tsx

```diff
@@ -339,10 +347,27 @@ class App extends Component<IAppProps, IAppState> {
     if (mobileScreen) {
       sidebarWidth = layout = 'col-xs-12 col-sm-12';
     }
-
+    if (permissionModeType && this.state.hideDialog) {
+      this.showDialog();
+    }
+    const teamsapp = <a href={"https://www.bing.com/?form=000010"}>Sample Explorer Teams app</a>;
```

Comment:
This will be changed later and will become a constant. I just put in a random link to make sure it works.